collection_manager/collection_manager/services/history_manager/SolrIngestionHistory.py

```diff
@@ -100,12 +99,11 @@ def _create_collection_if_needed(self):
                            'name': self._granule_collection_name,
                            'numShards': node_number
                            }
-                result = self._req_session.get('/'.join([self._solr_url.strip("/"), 'admin', 'collections']),
-                                               params=payload)
+                result = self._req_session.get(f"{self._solr_url.strip('/')}/solr/admin/collections", params=payload)
                 response = result.json()
                 logger.info(f"solr collection created {response}")
                 # Update schema
-                schema_url = '/'.join([self._solr_url.strip('/'), self._granule_collection_name, 'schema'])
+                schema_url = f"{self._solr_url.strip('/')}/{self._granule_collection_name}/schema"
```

Comment:
We don't need to add the /solr string in the path here ? I just want to make sure...

Reply:
You're right, /solr did need to be added to this.